### ISSUE TITLE
Ouverture d'une trame dans un nouvel onglet

### ISSUE THREAD

**Issue by vbdx** [kolekti/kolekti#17]
Dans l'explorateur des trames, quand on fait sur le nom d'une trame un clic droit + "Ouvrir le lien dans un nouvel onglet", seul le texte brut de la trame s'affiche dans le nouvel onglet

**Comment by kolekti** [comment]
oui, j'ai pensé que le click droit dans les explorateur des écrans kolekti donnent accès aux ressources "brutes" que l'on peut télécharger en faisant enregistrer sous.. 
C'est un comportement commun a tous les explorateurs de kolekti

Ce comportement peut etre changé par ajout d'un "badge" lien direct (a trouver)

**Comment by vbdx** [comment]
non, pas la peine. C'est juste que j'ai la mauvaise habitude d'ouvrir des onglets partout ...
Et je peux visiblement continuer à le faire avec un "clic droit + Ouvrir ..." sur l'onglet Trames (puis clic gauche sur le nom de la trame)
Donc pas de badge à prévoir

**Comment by kolekti** [comment]
D'une manière générale, il est déconseillé d'ouvrir plein d'onglets, l'interface n'est pas forcément synchro entre les différentes fenetres du navigateur, par exempel si tu ouvres une trame et que tu modifies les critères dans une autre fenêtre, tu ne pourra pas filtrer sur les nouveaux critères avant de recharger la page?

Il me semble donc interessant de travailler l'interface de façon à inciter l'utilisateur à rester sur le meme écran
